app/settings.py

```diff
@@ -34,11 +34,13 @@
 # Check if we're running a local development server or in prod.
 if os.environ.get('SERVER_SOFTWARE', '').startswith('Development'):
     DEBUG = True
+    DEBUG_PROPAGATE_EXCEPTIONS = True
     # If DEBUG is True and ALLOWED_HOSTS is empty, Django permits localhost.
     ALLOWED_HOSTS = []
     SECURE_SSL_REDIRECT = False
 else:
     DEBUG = False
+    DEBUG_PROPAGATE_EXCEPTIONS = False
     ALLOWED_HOSTS = site_settings.PROD_ALLOWED_HOSTS
```

Comment:
just curious, is it worth checking the env variable for .startswith('Prod') before allowind PROD_ALLOWED_HOSTS?

Reply:
No. From a security standpoint we'd want to err the other direction (limit the allowed_hosts, set debug to false, etc. unless we're sure it's dev).